src/components/data-management/SamplesTableCells.jsx

```diff
@@ -127,25 +127,23 @@ const UploadCell = (props) => {
     }
   };
   const onDownload = () => {
-    downloadSampleFile(activeExperimentId, sampleUuid, uploadDetailsModalData.name, selectedTech);
+    downloadSampleFile(
+      activeExperimentId, sampleUuid, uploadDetailsModalData.fileCategory, selectedTech,
+    );
   };
 
-  const onUpload = (fileObject, retryUpload = false) => {
-    if (!uploadDetailsModalData) {
-      return;
-    }
+  const onRetry = () => {
+    const fileType = uploadDetailsModalData.fileCategory;
+
     // if retrying an upload we dont need to revalidate the file since it was done before
-    if (retryUpload) {
-      createAndUploadSampleFile(fileObject, activeExperimentId, sampleUuid, dispatch, selectedTech);
-    } else {
-      fileObjectToFileRecord(fileObject, selectedTech).then((newFile) => {
-        if (newFile.valid) {
-          createAndUploadSampleFile(newFile, activeExperimentId, sampleUuid, dispatch, selectedTech);
-        } else {
-          handleError('error', endUserMessages.ERROR_FILE_CATEGORY);
-        }
-      });
```

Comment:
This code was unreachable. The case this was handling was when we replaced a file. This feature was removed a few weeks ago if I'm not mistaken